AgregarElementoPosicionIndicada.java

```diff
@@ -0,0 +1,38 @@
+/*
+En un arreglo ordenado agregar un elemento en una posición adecuada de manera que siga ordenado
+ */
+import java.util.Scanner;
+
+public class AgregarElementoPosicionIndicada {
+    public static void main(String[] args) {
+        int[] a = new int[10];
+        int[] b = new int[a.length+1];
+        int num, posicion=0;
+        Scanner s = new Scanner(System.in);
+
+        for (int i = 0; i < a.length; i++) {
+            System.out.print("Ingrese 10 números enteros: ");
+            a[i] = s.nextInt();
+        }
+        System.out.println("Arreglo: ");
+        for(int i = 0; i < a.length; i++) {

```

Comment:
Vamos a agregar una línea en blanco entre bloques como `for` `while` `if`, etc. Abajo y arriba.